src/composables/sessionManagement.js

```diff
@@ -0,0 +1,67 @@
+import { generatePeerIdCharacter } from "./peerIdGenerator";
+
+// In event order =>
+function storeSessionData(users) {
+  localStorage.setItem("users", JSON.stringify(users));
+}
+
+function getSessionData() {
+  const users = JSON.parse(localStorage.getItem("users"));
+  return users;
+}
+
+function startTimer(joinTime) {
```

Comment:
Why is there a join time, if a new time is initialized in the function

Reply:
The join time is when the a user joins the session